script.js

```diff
@@ -0,0 +1,49 @@
+var clickMe = document.querySelector('#first_paragraph');
+clickMe.addEventListener('click', function (){
+  console.log('h1 got clicked !!!!');
+  document.body.style.background = "blue";
+})
+
+var clickMe = document.querySelector('second_paragraph');
+clickMe.addEventListener('click', function (){
+  console.log('h2 got clicked!!!!');
+  document.body.style.background = "pink";
+})
+
+var clickMe = document.querySelector('first_chorus');
```

Comment:
Be careful about reusing the same variable name unless you really mean to. Having more specific variable names will probably be helpful.
```js
var firstChorus = document.querySelector('first_chorus'); //....etc
```